.github/workflows/validation_tests.yml

```diff
@@ -0,0 +1,105 @@
+name: Connector Ops CI - Run Live Tests
+
+concurrency:
+  # This is the name of the concurrency group. It is used to prevent concurrent runs of the same workflow.
+  #
+  # - github.head_ref is only defined on PR runs, it makes sure that the concurrency group is unique for pull requests
+  #  ensuring that only one run per pull request is active at a time.
+  #
+  # - github.run_id is defined on all runs, it makes sure that the concurrency group is unique for workflow dispatches.
+  # This allows us to run multiple workflow dispatches in parallel.
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+on:
+  workflow_dispatch:
+    inputs:
+      connector_name:
+        description: Connector name (e.g. source-faker)
+        required: true
+      connection_id:
+        description: ID of the connection to test; use "auto" to let the connection retriever choose a connection
+        required: true
+      pr_url:
+        description: URL of the PR containing the code change
+        required: true
+      streams:
+        description: Streams to include in tests
+      use_local_cdk:
+        description: Use the local CDK when building the target connector
+        default: "false"
+        type: boolean
+
+jobs:
+  live_tests:
+    name: Live Tests
+    runs-on: connector-test-large
+    timeout-minutes: 360 # 6 hours
+    steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@v4
+      - name: Check PAT rate limits
+        run: |
+          ./tools/bin/find_non_rate_limited_PAT \
+            ${{ secrets.GH_PAT_BUILD_RUNNER_OSS }} \
+            ${{ secrets.GH_PAT_BUILD_RUNNER_BACKUP }}
```

Comment:
In it's current state this worklow can't run on fork as:
* It is triggered by worfklow dispatch event, the checked out repo is the airbyte one
* There's no input to make this worfklow run on a different repo

I think it's fair to port it to the community_ci.yml workflow once we validate it can automatically run these test suite on our own PRs. Which is not the scope of this current change.